### PR TITLE
Adding manifest v11

### LINKED ISSUE #4205: bump manifest schema to v11 dbt-core PR #8335
At a minimum, update body to include a link to the page on docs.getdbt.com requiring updates and what part(s) of the page you would like to see updated.

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding Manifest v11 to the tables

Changed the manifest version table to a snippet.

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/snippets/_manifest-versions.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+| dbt Core version | Manifest version                                              |
+|------------------|---------------------------------------------------------------|
+| v1.7             | v11                                                           |
```
Comment (matthewshaver) [inline website/snippets/_manifest-versions.md]:
```suggestion
| v1.7             | [v11](https://schemas.getdbt.com/dbt/manifest/v11/index.html)     |
```

File: website/snippets/_manifest-versions.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+| dbt Core version | Manifest version                                              |
+|------------------|---------------------------------------------------------------|
+| v1.7             | [v11](https://schemas.getdbt.com/dbt/manifest/v11/index.html)     |
```
Comment (matthewshaver) [inline website/snippets/_manifest-versions.md]:
```suggestion
| v1.7             | [v11](https://schemas.getdbt.com/dbt/manifest/v11/index.html)  |
```

File: website/snippets/_manifest-versions.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+| dbt Core version | Manifest version                                              |
+|------------------|---------------------------------------------------------------|
+| v1.7             | [v11](https://schemas.getdbt.com/dbt/manifest/v11/index.html)  |
```
Comment (runleonarun) [inline website/snippets/_manifest-versions.md:4]: This manifest link is not working. Maybe it's just not published yet?

File: website/docs/reference/dbt-jinja-functions/model.md [hunk 4]
```diff
@@ -52,15 +52,9 @@ To view the structure of `models` and their definitions:
...
-| ----------- | ---------------- |
-| `v1.5` | [Manifest v9](https://schemas.getdbt.com/dbt/manifest/v9/index.html)
-| `v1.4` | [Manifest v8](https://schemas.getdbt.com/dbt/manifest/v8/index.html)
-| `v1.3` | [Manifest v7](https://schemas.getdbt.com/dbt/manifest/v7/index.html)
-| `v1.2` | [Manifest v6](https://schemas.getdbt.com/dbt/manifest/v6/index.html)
-| `v1.1` | [Manifest v5](https://schemas.getdbt.com/dbt/manifest/v5/index.html)
-
+import ManifestVersions from '/snippets/_manifest-versions.md';
 
+<ManifestVersions />
```
Comment (runleonarun) [inline website/docs/reference/dbt-jinja-functions/model.md:57]: Love me a good snippet! 🙇🏻

**Review by runleonarun — APPROVED** [review]
@matthewshaver The v11 manifest link is not working. Approving in case this is a known issue as I didn't see anyting wrong with the link itself.

**Pull request merged**